src/main/java/core/basesyntax/Application.java

```diff
@@ -2,7 +2,22 @@
 
 public class Application {
     public static void main(String[] args) {
-        // test your code here ...
+        Engine engine = new Engine();
+        engine.setVolume(2.0);
+        engine.setWeight(500);
+        engine.setNumber(1999199);
+        engine.setPower(250);
+        engine.setManufacturer("Bayerische Motoren Werke");
 
+        Car car = new Car();
+        car.setEngineOfCar(engine);
+        car.setMark("BMW");
+        car.setMaxSpeed(250);
+        car.setColor("Black");
+        car.setWeight(2000);
+
+        Car carClone = (Car) car.clone();
+        carClone.getEngineOfCar().setManufacturer("Munich Motoren Werke");
```

Comment:
it would be more informative to see the result of changing the engine of the original car - not the clone